scripts/src/main/resources/scripts/command/aws

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call aws with specified arguments.
+function doRun() {
+  doSetup silent
+  doRunCommand "${AWS_HOME}/aws" "${@}"
+}
+
+# $1: path_to_package
+# $2: enable_ide_update
+function doPackageInstall()
+{
+  local path_to_package="${1}"
+  local enable_ide_update="${2}"
+  local version_file="${path_to_package}/.devon.software.version"
+  local current_version
+  current_version="$(cat "${version_file}")"
+
+  if doIsWindows
+  then
+    doEcho "Windows Installation..."
+    windows_path_to_package=$(cygpath -w "${path_to_package}")
```

Comment:
This is not working outside cygwin (e.g. in git-bash what is our main shell on windows).
Please talk with the team - I left analog review comments in other PRs. We should add a function that converting the current path (or a given path as $1) to the native OS path depending on OS (mainly only special for Windows) and current shell (differs for cygwin, powershell, and MinGW/git-bash).

Reply:
as discussed, it works like this thanks to a recent Git Bash tweak

Reply:
Indeed. I was wrong. Great that this also works in git-bash. Thanks for clarification!